src/tests/unit/tests/DetailsView/components/requirement-view-title.test.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    RequirementViewTitle,
+    RequirementViewTitleProps,
+} from 'DetailsView/components/requirement-view-title';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+describe('RequirementViewTitleTest', () => {
+    it('renders with content from props', () => {
+        const props = {
```

Comment:
Should you add some stubs for guidance links?

Reply:
any props that are being used should be here and stubbed out.

Reply:
Added props stubs